R/maicplus.R

```diff
@@ -0,0 +1,30 @@
+#' maicplus: open source R package for MAIC
+#'
+#' A package for running matching adjusted indirect comparison
+#'
+#' This package facilitates performing matching adjusted indirect comparison
+#' (MAIC) analysis where the endpoint of interest is either time-to-event
+#' (e.g. overall survival) or binary (e.g. objective tumor response).
+#'
+#' @docType package
+#' @name maicplus-package
```

Comment:
Suggest to move this to DESCRIPTION and then use roxygen to pick it up automatically

```suggestion
#' @keywords internal
"_PACKAGE"

# The following block is used by usethis to automatically manage
# roxygen namespace tags. Modify with care!
## usethis namespace: start
## usethis namespace: end
NULL
```

eg in DESCRIPTION it could be
```
Title: Matching adjusted indirect comparison
Description: The maicplus package facilitates performing matching adjusted indirect comparison
 (MAIC) analysis where the endpoint of interest is either time-to-event
 (e.g. overall survival) or binary (e.g. objective tumor response).
```